dev/postgres/9.5/check.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
```

Comment:
> but I saw someone complaining about 9.5 in an issue somewhere

It's supported in Sequelize 6 (because dropping support is a breaking change), but not in v7 (which is what this branch is based on)